gui/widget.h

```diff
@@ -476,6 +476,15 @@ class OptionsContainerWidget : public Widget {
 	 */
 	virtual bool save() = 0;
 
+	/** Implementing classes should return if there are relevant keys set in the configuration domain
+	 *
+	 * @return true if there relevant are keys set in the configuration domain
```

Comment:
This is a minor thing, but the comment needs a syntax correction to:
"true, if there are relevant keys set in the configuration domain"
as in the line above.

Reply:
Fixed, thanks.